src/Arcus.Scripting.DevOps/Arcus.Scripting.DevOps.psm1

```diff
@@ -98,10 +98,11 @@ Export-ModuleMember -Function Set-AzDevOpsArmOutputsToPipelineVariables
 function Save-AzDevOpsBuild {
     param(        
         [Parameter(Mandatory = $true)][string] $ProjectId = $(throw "ProjectId is required"),
-        [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required")
+        [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required"),
+        [Parameter(Mandatory = $false)][int] $DaysToKeep
```

Comment:
Is the default zero when not passed in?

Reply:
Indeed, and if it is set to 0 we'll change it to 99999 to indicate "forever"